usaspending_api/accounts/v2/filters/account_download.py

```diff
@@ -318,6 +318,46 @@ def award_financial_derivations(derived_fields):
         "award__latest_transaction__contract_data__funding_office_name",
         "award__latest_transaction__assistance_data__funding_office_name",
     )
+    derived_fields["recipient_duns"] = Coalesce(
+        "award__latest_transaction__contract_data__awardee_or_recipient_uniqu",
+        "award__latest_transaction__assistance_data__awardee_or_recipient_uniqu",
+    )
+    derived_fields["recipient_name"] = Coalesce(
+        "award__latest_transaction__contract_data__awardee_or_recipient_legal",
+        "award__latest_transaction__assistance_data__awardee_or_recipient_legal",
+    )
+    derived_fields["recipient_parent_duns"] = Coalesce(
+        "award__latest_transaction__contract_data__ultimate_parent_unique_ide",
+        "award__latest_transaction__assistance_data__ultimate_parent_unique_ide",
+    )
+    derived_fields["recipient_parent_name"] = Coalesce(
+        "award__latest_transaction__contract_data__ultimate_parent_legal_enti",
+        "award__latest_transaction__assistance_data__ultimate_parent_legal_enti",
+    )
+    derived_fields["recipient_country"] = Coalesce(
+        "award__latest_transaction__contract_data__legal_entity_country_code",
+        "award__latest_transaction__assistance_data__legal_entity_country_code",
+    )
+    derived_fields["recipient_state"] = Coalesce(
+        "award__latest_transaction__contract_data__legal_entity_state_code",
+        "award__latest_transaction__assistance_data__legal_entity_state_code",
+    )
+    derived_fields["recipient_county"] = Coalesce(
+        "award__latest_transaction__contract_data__legal_entity_county_name",
+        "award__latest_transaction__assistance_data__legal_entity_county_name",
+    )
+    derived_fields["recipient_city"] = Coalesce(
+        "award__latest_transaction__contract_data__legal_entity_city_name",
+        "award__latest_transaction__assistance_data__legal_entity_city_name",
+    )
+    derived_fields["recipient_congressional_district"] = Coalesce(
+        "award__latest_transaction__contract_data__legal_entity_congressional",
+        "award__latest_transaction__assistance_data__legal_entity_congressional",
+    )
+    derived_fields["recipient_zip_code"] = Coalesce(
```

Comment:
Appears the current downloads pull the "zip+4" 
`award__recipient__location__zip4` in usaspending_api/download/v2/download_column_historical_lookups.py

Reply:
On the data dictionary (Data Transparency Rosetta Stone Full), line 223 it says "legal_entity_zip5,
legal_entity_zip5,
zip5". Contradiction?

Reply:
Even if so, that bug would be out of scope of this work. I'll set it to legacy behavior.

Reply:
Seems like we're both wrong. FPDS and FABS have different ZIP fields.

FPDS has a combined ZIP+4 while FABS has ZIP5 and ZIP last4 fields. Currently the downloads use {{references_location}} which contains both ZIP and ZIP+4 values in the column.  I didn't investigate all the way to understand exactly how they are stored for each data source. "Legacy behavior" might be just as difficult as the optimal solution depending on the buried logic